app/test_main.py

```diff
@@ -1,3 +1,34 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_get_human_age_zero() -> None:
```

Comment:
Consider using `@pytest.mark.parametrize` to simplify and consolidate similar test cases. This will help avoid repetition and make the tests more maintainable, as suggested in the checklist.